### ISSUE TITLE
reference best

### ISSUE THREAD

**Issue by DorisAmoakohene** [DorisAmoakohene/Codes-and-Test-for-my-paper#5]
Hi @tdhock if you say "please remove facet_grid and instead use a Y axis" do you mean like the below 
![best list R](https://github.com/user-attachments/assets/de3b93e1-d456-4ee3-aa2e-5e66cff3d20a)

**Comment by tdhock** [comment]
yes that is better but now please remove aes(color) in geom_ribbon
and please use direct labels instead of legend
also remove ".replacement" from expression names

**Comment by DorisAmoakohene** [comment]
![best list R](https://github.com/user-attachments/assets/5ed28840-5e1d-4dcf-b2ec-d906765f592d)

**Comment by tdhock** [comment]
hi Doris that is better, but please remove tick marks outside the range of the data (10000)
also it may be better to increase N, so that both go up to some time limit, say 0.1 or 1 second?
also you may want to re-run the timings because the curves are somewhat choppy, which is unusual. Typically I would expect them to be a bit more smooth, unless the data are random? and even then you should be using set.seed to control the randomness and reduce the variability between N values.

**Comment by DorisAmoakohene** [comment]
![best list R](https://github.com/user-attachments/assets/b5e7ffe1-0a56-4eb2-be6b-f011fd08d146)

**Comment by tdhock** [comment]
looks good!

**Comment by tdhock** [comment]
although "linear" and "constant" may be confusing names because the reader would think about linear time and constant time, but actually "constant" label is linear time, and "linear" label is quadratic time, so maybe think about changing the labels, or using a different example, where there is no such confusion?

**Comment by DorisAmoakohene** [comment]
I will change the labels then. Thanks for the clarity
